src/components/StaffList/index.jsx

```diff
@@ -10,10 +12,56 @@ class StaffList extends React.Component {
     super(props);
     this.state = {
       staffList: props.input.value || [], // requires staffList to be an array
+      suggestions: [], // the suggestions returned by the discovery service for autocomplete
+      searchString: '', // search input provided by the user
     };
 
     this.handleRemove = this.handleRemove.bind(this);
     this.onDragEnd = this.onDragEnd.bind(this);
+    this.onChange = this.onChange.bind(this);
+    this.onSuggestionsFetchRequested = this.onSuggestionsFetchRequested.bind(this);
+    this.onSuggestionsClearRequested = this.onSuggestionsClearRequested.bind(this);
+    this.getSuggestionValue = this.getSuggestionValue.bind(this);
+    this.renderSuggestion = this.renderSuggestion.bind(this);
+  }
+
+  onChange(event, { newValue }) {
```

Comment:
nit: I don't think you need the event param

Reply:
👍